tests/test_create_2.py

```diff
@@ -0,0 +1,26 @@
+import utilities.methods
```

Comment:
Odnośnie importów to PyCharm gdy zaczęłam rozdzielać i importować tylko używane metody w teście polecił mi importowanie całego modułu?

Reply:
faktycznie, trzeba to pewnie zmienić w ustawieniach PyCharma:
tutaj wersja dla IntelliJ, ale szukałabym czegoś podobnego pod hasłem 'wildcard'
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports